runelite-client/src/main/java/net/runelite/client/plugins/barrows/CryptLocations.java

```diff
@@ -0,0 +1,265 @@
+			new WorldPoint(3574, 3281, 0),
+			new WorldPoint(3574, 3282, 0),
+			new WorldPoint(3574, 3283, 0),
+			new WorldPoint(3574, 3284, 0),
+			new WorldPoint(3575, 3280, 0),
+			new WorldPoint(3575, 3281, 0),
+			new WorldPoint(3575, 3282, 0),
+			new WorldPoint(3575, 3283, 0),
+			new WorldPoint(3575, 3284, 0),
+			new WorldPoint(3576, 3280, 0),
+			new WorldPoint(3576, 3281, 0),
+			new WorldPoint(3576, 3282, 0),
+			new WorldPoint(3576, 3283, 0),
+			new WorldPoint(3576, 3284, 0),
+			new WorldPoint(3577, 3280, 0),
+			new WorldPoint(3577, 3281, 0),
+			new WorldPoint(3577, 3282, 0),
+			new WorldPoint(3577, 3283, 0),
+			new WorldPoint(3577, 3284, 0),
+			new WorldPoint(3578, 3280, 0),
+			new WorldPoint(3578, 3281, 0),
+			new WorldPoint(3578, 3282, 0),
+			new WorldPoint(3578, 3283, 0),
+			new WorldPoint(3578, 3284, 0),
+			new WorldPoint(3579, 3280, 0),
+			new WorldPoint(3579, 3281, 0),
+			new WorldPoint(3579, 3282, 0),
+			new WorldPoint(3579, 3283, 0),
+			new WorldPoint(3579, 3284, 0),
+			new WorldPoint(3580, 3280, 0),
+			new WorldPoint(3580, 3281, 0),
+			new WorldPoint(3580, 3282, 0),
+			new WorldPoint(3580, 3283, 0),
+			new WorldPoint(3580, 3284, 0)),
+	TORAGS(new WorldPoint(3552, 3281, 0),
+			new WorldPoint(3552, 3282, 0),
+			new WorldPoint(3552, 3283, 0),
+			new WorldPoint(3552, 3284, 0),
+			new WorldPoint(3553, 3281, 0),
+			new WorldPoint(3553, 3282, 0),
+			new WorldPoint(3553, 3283, 0),
+			new WorldPoint(3553, 3284, 0),
+			new WorldPoint(3554, 3281, 0),
+			new WorldPoint(3554, 3282, 0),
+			new WorldPoint(3554, 3283, 0),
+			new WorldPoint(3554, 3284, 0),
+			new WorldPoint(3555, 3281, 0),
+			new WorldPoint(3555, 3282, 0),
+			new WorldPoint(3555, 3283, 0),
+			new WorldPoint(3555, 3284, 0),
+			new WorldPoint(3556, 3281, 0),
+			new WorldPoint(3556, 3282, 0),
+			new WorldPoint(3556, 3283, 0),
+			new WorldPoint(3556, 3284, 0)),
+	VERACS(new WorldPoint(3555, 3295, 0),
+			new WorldPoint(3555, 3296, 0),
+			new WorldPoint(3555, 3297, 0),
+			new WorldPoint(3555, 3298, 0),
+			new WorldPoint(3555, 3299, 0),
+			new WorldPoint(3555, 3300, 0),
+			new WorldPoint(3556, 3295, 0),
+			new WorldPoint(3556, 3296, 0),
+			new WorldPoint(3556, 3297, 0),
+			new WorldPoint(3556, 3298, 0),
+			new WorldPoint(3556, 3299, 0),
+			new WorldPoint(3556, 3300, 0),
+			new WorldPoint(3557, 3295, 0),
+			new WorldPoint(3557, 3296, 0),
+			new WorldPoint(3557, 3297, 0),
+			new WorldPoint(3557, 3298, 0),
+			new WorldPoint(3557, 3299, 0),
+			new WorldPoint(3557, 3300, 0),
+			new WorldPoint(3558, 3295, 0),
+			new WorldPoint(3558, 3296, 0),
+			new WorldPoint(3558, 3297, 0),
+			new WorldPoint(3558, 3298, 0),
+			new WorldPoint(3558, 3299, 0),
+			new WorldPoint(3558, 3300, 0),
+			new WorldPoint(3559, 3295, 0),
+			new WorldPoint(3559, 3296, 0),
+			new WorldPoint(3559, 3297, 0),
+			new WorldPoint(3559, 3298, 0),
+			new WorldPoint(3559, 3299, 0),
+			new WorldPoint(3559, 3300, 0)),
+	KARILS(new WorldPoint(3563, 3273, 0),
```

Comment:
This code might benefit from creating a `WorldRect` structure which takes in a origin and size and can output all the points inside.

Reply:
Could probably just add a method to `WorldArea`

Reply:
Maybe a constructor that takes in two tiles? Northwest and Southeast? that should be enough